packages/sdk/src/rpc/rpc.ts

```diff
@@ -204,11 +204,15 @@ export class Rpc {
   /**
    * Broadcast a Tx to the ledger
    * @async
-   * @param {SignedTx} signedTx - Transaction with signature
-   * @returns {void}
+   * @param signedTx - Transaction with signature
+   * @returns void
```

Comment:
Now returns string instead of void.

Reply:
This unfortunately won't be caught by jsdoc plugin :(

Reply:
I changed return description to optional :)